src/components/Button.jsx

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components"
+
+export const Button = styled.button`
+  background: ${props => (props.primary ? "#4159de" : "hite")};
+  color: ${props => (props.primary ? "#f7fafc" : "alevioletred")};
+
+  font-size: 14xpx;
+  margin: 1em;
+  padding: 0.7rem 2em;
+  border: none;
+  border-radius: 1px;
```

Comment:
This button doesn't have a `border-radius`, it's a straight rectangle. You can remove this property.